src/main/java/org/sagebionetworks/web/client/presenter/AccessRequirementPresenter.java

```diff
@@ -59,37 +60,44 @@ public void setPlace(AccessRequirementPlace place) {
     String id = place.getParam(AccessRequirementsPlace.ID_PARAM);
     String typeString = place.getParam(AccessRequirementsPlace.TYPE_PARAM);
 
-    // Note: configuring the Access Requirement widget without a target subject will result in notifications sent to the user will not have the context (Project/Folder/File associated with the restriction).
```

Comment:
This diff is terrible.  Basically, this component now always queries for the AccessRequirement (used to only do this if the subject was not given).  The view title is now based on the Access Requirement name (which did not exist when this presenter was written!).  Also, we can't assume that the subject array has at least one element (with the new annotation based subjects).